.github/actions/prepare/action.yml

```diff
@@ -8,7 +8,7 @@ runs:
     - uses: actions/setup-node@v4
       with:
         cache: pnpm
-        node-version: "18"
+        node-version: "22"
```

Comment:
> I updated the prepare action to use node 22, which will allows to use the Object.groupBy api (introduced in v21). Since 22 is now LTS, I though that was probably ok? But if you'd rather leave it on node 18, I'm happy to refactor that.

For context, the reason I'd kept it on 18 even after 20 went LTS is to make sure scripts that actually run the code don't crash on older Node.js versions. Specifically Node.js APIs aren't checked by TypeScript the way globals or syntax are. But CTA repos now include lint rules like [`n/no-unsupported-features/es-builtins`](https://github.com/eslint-community/eslint-plugin-n/blob/86a524250dcc7c32225f2880ec66767a06c6258d/docs/rules/no-unsupported-features/es-builtins.md). So I think this is fine to switch for faster Node.js performance + more built-ins. 🚀